scripts/server

```diff
@@ -12,17 +12,26 @@ Starts servers using docker-compose
 "
 }
 
-
 if [ "${BASH_SOURCE[0]}" = "${0}" ]
 then
     if [ "${1:-}" = "--help" ]
     then
         usage
+    elif [ "$1" = "--no-airflow" ]
```

Comment:
It seems as though the more common case is that people will be working without Airflow vs. with it. I think it might make sense to flip this conditional so that `--with-airflow` triggers the launching of the Airflow services.